libatik/CMakeLists.txt

```diff
@@ -1,8 +1,8 @@
 cmake_minimum_required(VERSION 3.0)
 PROJECT(libatik C CXX)
 
-# Built Jan 19th, 2020
-set(ATIK_VERSION "2.2.6")
+# Built May 6th, 2020
+set(ATIK_VERSION "1.4.5")
```

Comment:
Here I used the same version as the API/DLL version from the SDK. I'm not sure why it was 2.2.6 before, except to match the INDI driver version. But that should be the SDK version actually. Opinion?

Reply:
This is wrong. Driver's version should be set to 2.2.7

Reply:
Except this is not the driver, this is the SDK library bundle. The driver version is in `indi-atik/CMakeLists.txt`. Unless there is a good reason that I don't know about, this should reflect the API/DLL version, which is returned as `145` by the Artemis API.

Reply:
I think it is legacy versioning. All the other libs use SDK version number. So in general you are right. However as for now it is aligned with version in debian packaging system. If we just decrease version, users will not get the updated libatik (1.4.5 << 2.2.7). Therefore my suggestion is to release the libatik as 2.2.7 and set(ATIK_VERSION "2.13.3.31") based on  libflycapture.so.2.13.3.31. This will be consistent how we version other libs.

Reply:
Done.